CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/value/expression/ExpressionCompiler.java

```diff
@@ -89,35 +89,38 @@ private ExpressionCompiler() {
+        final Object left = compileExpression(children.get(0), variableProvider, indexReadable, operators);
+        final OperatorExpression operator = (OperatorExpression) children.get(1);
+        final Object right = compileExpression(children.get(2), variableProvider, indexReadable, operators);
+        final String operatorName = OPERATOR_CLASSES.get(operator.getOperator());
+        final Object result = operators.getRegistry(operatorName).apply(left, right);
+        if (result == null) {
+            throw new ExpressionException("Unable to find implementation of operator: " + operatorName + " (" + left.getClass().getCanonicalName() + ", " + right.getClass().getCanonicalName() + ")");
+        }
+        return result;
+    }
+
     public static Object compileSequenceExpression(SequenceExpression expression, VariableProvider variableProvider, IntReadable indexReadable, Operators operators) {
         final List<Expression> children = expression.getUnmodifiableChildren();
         switch (children.size()) {
             case 1:
                 return compileExpression(children.get(0), variableProvider, indexReadable, operators);
-
-            case 2: {
-                final OperatorExpression operator = (OperatorExpression) children.get(0);
-                final Object right = compileExpression(children.get(1), variableProvider, indexReadable, operators);
-                final String operatorName = CONVERTER_CLASSES.get(operator.getOperator());
-                final Object result = operators.getRegistry(operatorName).apply(right);
-                if (result == null) {
-                    throw new ExpressionException("Unable to find implementation of operator: " + operatorName + " (" + right.getClass().getCanonicalName() + ")");
-                }
-                return result;
-            }
-
-            case 3: {
-                final Object left = compileExpression(children.get(0), variableProvider, indexReadable, operators);
-                final OperatorExpression operator = (OperatorExpression) children.get(1);
-                final Object right = compileExpression(children.get(2), variableProvider, indexReadable, operators);
-                final String operatorName = OPERATOR_CLASSES.get(operator.getOperator());
-                final Object result = operators.getRegistry(operatorName).apply(left, right);
-                if (result == null) {
-                    throw new ExpressionException("Unable to find implementation of operator: " + operatorName + " (" + left.getClass().getCanonicalName() + ", " + right.getClass().getCanonicalName() + ")");
-                }
-                return result;
-            }
-
+            case 2:
+                return compileTwoChildrenExpression(children, variableProvider, indexReadable, operators);
```

Comment:
I'll just point out as an FYI that an easier solution for the code smell you were fixing here would've been to simply omit the braces as they were unnecessary in the switch statement case

Reply:
I use them myself and didn't know that you could do multi-line without them. I can revert adding them to methods if it helps readability?